source/guides/style_guide.markdown

```diff
@@ -137,7 +137,7 @@ Every publicly available module must have metadata defined in the metadata.json
 ~~~
     {
       "name": "examplecorp-mymodule",
-      "version": "0.0.1",
+      "version": "0.1.0",
```

Comment:
This is here because versioning starts at 0.1.0, so 0.0.1 is not possible with semver.